apps/self-hosted/src/styles/blog-markdown.css

```
@@ -1321,7 +1327,24 @@
+ * `:focus-within` is reached by keyboard, and a touch device with a hardware
+ * keyboard reports `hover: none`, so gating it would take the reveal away from
+ * exactly the people who navigate that way. Only the pointer half belongs
+ * inside the guard.
+ */
+@media (hover: hover) {
+  .markdown-body .contains-task-list:hover .task-list-item-convert-container {
+    display: block;
+    width: auto;
+    height: 24px;
+    overflow: visible;
+    clip: auto;
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)